### PR TITLE
ci(operator): add long longevities CI jobs

### PR DESCRIPTION
**Pull request by vponomaryov**
List of new jobs:
- 200gb-48h
- large-collections-12h
- lwt-24h
- twcs-48h

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Review by enaydanov — DISMISSED** [review]

File: jenkins-pipelines/operator/eks/longevity-scylla-operator-200gb-48h.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-200GB-48h-verifier-LimitedMonkey-tls.yaml", "configurations/operator/200GB-no-tls.yaml"]''',
+    email_recipients: 'qa@scylladb.com,scylla-operator@scylladb.com',
+    k8s_enable_performance_tuning: true,
+    availability_zone: 'a,b',
+    post_behavior_db_nodes: 'destroy',
+    post_behavior_loader_nodes: 'destroy',
+    post_behavior_monitor_nodes: 'destroy',
+    post_behavior_k8s_cluster: 'destroy',
+    timeout: [time: 3060, unit: 'MINUTES']
```
Comment (bentsi) [inline jenkins-pipelines/operator/eks/longevity-scylla-operator-200gb-48h.jenkinsfile:18]: not needed, timeouts are calculated in getJobTimeouts.groovy

File: jenkins-pipelines/operator/eks/longevity-scylla-operator-lwt-24h.jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    test_name: 'longevity_lwt_test.LWTLongevityTest.test_lwt_longevity',
+    test_config: '''["test-cases/longevity/longevity-lwt-basic-24h.yaml", "configurations/operator/lwt-basic-24h.yaml"]''',
+    email_recipients: 'qa@scylladb.com,scylla-operator@scylladb.com',
+    k8s_enable_performance_tuning: true,
+    availability_zone: 'a,b',
+    post_behavior_db_nodes: 'destroy',
+    post_behavior_loader_nodes: 'destroy',
+    post_behavior_monitor_nodes: 'destroy',
+    post_behavior_k8s_cluster: 'destroy',
+    timeout: [time: 2000, unit: 'MINUTES']
```
Comment (bentsi) [inline jenkins-pipelines/operator/eks/longevity-scylla-operator-lwt-24h.jenkinsfile:18]: not needed, timeouts are calculated in getJobTimeouts.groovy

File: jenkins-pipelines/operator/eks/longevity-scylla-operator-twcs-48h.jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    test_name: 'longevity_twcs_test.TWCSLongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-twcs-48h.yaml", "configurations/operator/twcs-basic-48h.yaml"]''',
+    email_recipients: 'qa@scylladb.com,scylla-operator@scylladb.com',
+    k8s_enable_performance_tuning: true,
+    availability_zone: 'a,b',
+    post_behavior_db_nodes: 'destroy',
+    post_behavior_loader_nodes: 'destroy',
+    post_behavior_monitor_nodes: 'destroy',
+    post_behavior_k8s_cluster: 'destroy',
+    timeout: [time: 3060, unit: 'MINUTES']
```
Comment (bentsi) [inline jenkins-pipelines/operator/eks/longevity-scylla-operator-twcs-48h.jenkinsfile:18]: not needed, timeouts are calculated in getJobTimeouts.groovy

File: jenkins-pipelines/operator/eks/longevity-scylla-operator-large-collections-12h.jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: '''["test-cases/longevity/longevity-large-collections-12h.yaml", "configurations/operator/large-collections-12h.yaml"]''',
+    email_recipients: 'qa@scylladb.com,scylla-operator@scylladb.com',
+    k8s_enable_performance_tuning: true,
+    availability_zone: 'a,b',
+    post_behavior_db_nodes: 'destroy',
+    post_behavior_loader_nodes: 'destroy',
+    post_behavior_monitor_nodes: 'destroy',
+    post_behavior_k8s_cluster: 'destroy',
+    timeout: [time: 870, unit: 'MINUTES']
```
[the remark this answers is not in the document]
Comment (vponomaryov) [reply jenkins-pipelines/operator/eks/longevity-scylla-operator-large-collections-12h.jenkinsfile]: Done

File: jenkins-pipelines/operator/eks/longevity-scylla-operator-lwt-24h.jenkinsfile (hunk 2 above)
Comment (vponomaryov) [reply jenkins-pipelines/operator/eks/longevity-scylla-operator-lwt-24h.jenkinsfile]: Done

File: jenkins-pipelines/operator/eks/longevity-scylla-operator-twcs-48h.jenkinsfile (hunk 3 above)
Comment (vponomaryov) [reply jenkins-pipelines/operator/eks/longevity-scylla-operator-twcs-48h.jenkinsfile]: Done

**Review by bentsi — APPROVED** [review]

**Pull request merged**